gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -567,13 +568,15 @@ accept_socket_async_cb (GObject * source, GAsyncResult * result,
   saddr = g_socket_get_remote_address (socket, &err);
   if (!saddr) {
     nns_loge ("Failed to get socket address: %s", err->message);
+    g_free (conn);
```

Comment:
need to free `err`? line 558 as well.

Reply:
Right!. `g_clear_error()` should be needed after using `GError`.

Reply:
Thanks. Updated!